modules/contact/test/tests/mortar_cartesian_lms/tests

```diff
@@ -10,6 +10,7 @@
     max_parallel = 4
     min_ad_size = '50'
     valgrind = 'heavy'
+    installation_type = in_tree
```

Comment:
Why is this needed?

Reply:
This test does not seem to work when ran out of tree. That is, `./configure --prefix /some/path; make install`

Reply:
@lindsayad any more comment here? This package update is getting close to being ready to merge - just turned on the full complement of `next` testing.

Reply:
I just have no idea why this test should be different than any other. But it's definitely not important enough to me to prevent this moving forward

Reply:
Good idea about the "full `next` complement".

Reply:
This test is fine.

I think what happened is that I mistook a faulty test for an actual MPI version issue (which I redacted several commits ago).